steps/save-cache/1.1.1/step.yml

```diff
@@ -0,0 +1,118 @@
+
+  #### Related steps
+
+  [Restore cache](https://github.com/bitrise-steplib/bitrise-step-restore-cache/)
+website: https://github.com/bitrise-steplib/bitrise-step-save-cache
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-save-cache
+support_url: https://github.com/bitrise-steplib/bitrise-step-save-cache/issues
+published_at: 2023-02-10T08:49:37.205256304Z
+source:
+  git: https://github.com/bitrise-steplib/bitrise-step-save-cache.git
+  commit: f122ec1ec2d6de1262f49e1c6b03db45dcb1c1f3
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/bitrise-step-save-cache
+deps:
+  brew:
+  - name: zstd
+  apt_get:
+  - name: zstd
+is_skippable: true
+run_if: .IsCI
+inputs:
+- key: null
+  opts:
+    description: |-
+      Key used for saving a cache archive.
+
+      The key supports template elements for creating dynamic cache keys. These dynamic keys change the final key value based on the build environment or files in the repo in order to create new cache archives. See the Step description for more details and examples.
+
+      The maximum length of a key is 512 characters (longer keys get truncated). Commas (`,`) are not allowed in keys.
+    is_required: true
+    summary: Key used for saving a cache archive. This can contain template elements.
+    title: Cache key
+- opts:
+    description: |-
+      List of files and folders to include in the cache.
+
+      Add one path per line. Each path can contain wildcards (`*` and `**`) that are evaluated at runtime.
+    is_required: true
+    summary: List of files and folders to include in the cache.
+    title: Paths to cache
+  paths: null
+- opts:
+    is_required: true
+    summary: Enable logging additional information for troubleshooting
+    title: Verbose logging
+    value_options:
+    - "true"
+    - "false"
+  verbose: "false"
+- is_key_unique: "false"
+  opts:
+    description: |-
+      Enabling this allows the Step to skip creating a new cache archive when the workflow previously restored the cache with the same key.
+
+      This requires the cache key to be unique, so that the key changes whenever the files in the cache change. In practice, this means adding a `checksum` part to the key template with a file that describes the cache content (such as a lockfile).
+
+      Example of a cache key where this can be safely turned on: `npm-cache-{{ checksum "package-lock.json" }}`. On the other hand, `my-cache-{{ .OS }}-{{ .Arch }}` is not unique (even though it uses templates).
+
+      Note: the Step can still skip uploading a cache when this input is `false`, it just needs to create the archive first to compute its checksum (which takes time).
+    summary: This allows the Step to skip creating a new cache in certain conditions.
+      Only set to `true` if the key is unique to the cached content!
+    title: Unique cache key
+    value_options:
+    - "true"
+    - "false"
```

Comment:
The PR comment says that this is new addition. Is not this a breaking change? The field could contain any value because it was not a value options but from now on only the `true` and `false` values are accepted.

Reply:
I think it would be in the other direction, but not a breaking change in this direction. So far this input has been a `string` type (accidentally), but since every input is a string under the hood, even the new `true` and `false` values are strings too. Of course, if someone defined any other value here previously, this will now fail at the time of parsing the step, but it also fails now at a later stage, when the step code parses the inputs into the `Config` structure.

Reply:
I just tested this by replacing the currently available `save-cache` step with the Git URL of the latest state in one workflow and it didn't break anything